sdk/src/wallet/account/operations/participation/mod.rs

```diff
@@ -284,7 +284,6 @@ where
             .get_participation_events(account_index)
             .await?;
 
-        // TODO try to remove this clone
```

Comment:
I tried and couldn't find a better way since it contains async code we can't use retain

Reply:
Yeah I tried a lot as well and found nothing satisfying